config/webpack/test.webpack.config.js

```diff
@@ -132,16 +134,16 @@ function getWebpackConfig(skyPagesConfig) {
         'ROOT_DIR': JSON.stringify(srcPath),
         'SKY_PAGES': JSON.stringify(skyPagesConfig)
       }),
-    ],
 
-    tslint: {
-      emitErrors: false,
-      failOnHint: false,
-      resourcePath: 'src'
-    },
+      new ContextReplacementPlugin(
+        // The (\\|\/) piece accounts for path separators in *nix and Windows
+        /angular(\\|\/)core(\\|\/)(esm(\\|\/)src|src)(\\|\/)linker/,
+        skyPagesConfigUtil.spaPath('src') // location of your src
+      )
+    ],
 
     node: {
-      global: 'window',
+      global: true,
```

Comment:
Similar to the comment added in `serve.webpack.conf.js`, can we explore whether these are truly needed?